CourseGrab/ViewControllers/Search/SearchDetailTableViewController.swift

```diff
@@ -10,34 +10,46 @@ import UIKit
 
 class SearchDetailTableViewController: UITableViewController {
 
+    private enum CellIdentifier: String {
+
+        case card
+        case section
+
+    }
```

Comment:
oh this is interesting. typically i use make string constants for the cell identifiers at the top of the class and use those but this is pretty clean. wondering what others think of this